examples/ncurses/RF24Gateway_ncurses.cpp

```diff
@@ -735,3 +653,104 @@ void drawConnPad()
 }
 
 /******************************************************************/
+
+/** Handle keyboard input **/
+/*******************************/
+
+bool handleKeyPad()
+{
+
+    int myChar = getch();
+
+    if (myChar > -1)
+    {
+        //cout << myChar << endl;
+        switch (myChar)
+        {
+            // a: En/Disable display of active connections
+            case 'a':
+                if (topo) {
+                    showConnPad = true;
+                }
+                else {
+                    showConnPad = !showConnPad;
+                }
+                topo = false;
+                if (!showConnPad)
+                {
+                    werase(connPad);
+                    prefresh(connPad, 0, 0, 15, 1, maxX - 1, maxY - 2);
+                    drawMain();
+                }
+                break;
+            // w: Increase frame-rate of curses display
+            case 'w':
+                if (updateRate > 100)
+                {
+                    updateRate -= 100;
+                }
+                mvwprintw(win, 2, 27, "Refresh Rate: %.1f fps", 1000.0 / updateRate);
+                refresh();
+                break;
+            // s: Decrease frame-rate of curses display
+            case 's':
+                updateRate += 100;
+                mvwprintw(win, 2, 27, "Refresh Rate: %.1f fps   \t", 1000.0 / updateRate);
+                refresh();
+                break;
+            // c: Display IP configuration menu
+            case 'c':
+                drawCfg(1);
+                break;
+            // h: Display help menu
+            case 'h':
+                drawHelp();
+                break;
+            case 'x':
+                erase();
+                endwin();
+                return 0;
+                break;
+            case 'A':
+                if (padSelection == 0)
+                {
+                    meshScroll++;
+                }
+                else if (padSelection == 1)
+                {
+                    connScroll = rf24_max(connScroll++, 0);
+                }
+                break;
+            case 'B':
+                if (padSelection == 0)
+                {
+                    meshScroll--;
+                }
+                else if (padSelection == 1)
+                {
+                    connScroll--;
+                }
+                break;
+            case 'C':
+                padSelection++;
+                padSelection = std::min(padSelection, 1);
+                break; //right
+            case 'D':
+                padSelection--;
+                padSelection = std::max(padSelection, 0);
+                break; //left
+                meshScroll = std::max(meshScroll, 0);
+                connScroll = std::max(connScroll, 0);
+                meshInfoTimer = 0;
```

Comment:
still keeping this dead code in there?

Reply:
same for the interrupt example too

Reply:
Lol nope, just got rid of it. Thanks for pointing that out.